### ISSUE TITLE
Table: Not keyboard accessible

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-core#633]
For tables with selectable rows, it should be possible to arrow between the rows and use enter/space to select a row. Refer to OneDrive's file list as a reference.

![image](https://cloud.githubusercontent.com/assets/1382445/15807771/fca2b514-2b1a-11e6-9981-4c0b62f34423.png)

**Comment by mikewheaton** [comment]
This issue was moved to OfficeDev/office-ui-fabric-js#52
